feature/gribi/ate_tests/base_hierarchical_route_installation_test/base_hierarchical_route_installation_test.go

```diff
@@ -179,7 +179,7 @@ func applyForwardingPolicy(t *testing.T, ingressPort string) {
 	pfPath := gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).PolicyForwarding().Interface(ingressPort)
 	pfCfg := d.GetOrCreateNetworkInstance(deviations.DefaultNetworkInstance(dut)).GetOrCreatePolicyForwarding().GetOrCreateInterface(ingressPort)
 	pfCfg.ApplyVrfSelectionPolicy = ygot.String(policyName)
-	if deviations.ExplicitInterfaceRefDefinition(dut) {
+	if !deviations.InterfaceRefConfigUnsupported(dut) {
```

Comment:
I think we should try to avoid double negative to improve readability. 

Is it better to write something like the following?

```
pfCfg.GetOrCreateInterfaceRef().Interface = ygot.String(ingressPort)
pfCfg.GetOrCreateInterfaceRef().Subinterface = ygot.Uint32(0)
if deviations.InterfaceRefConfigUnsupported(dut) {
   // unset the interface ref
}
```

Reply:
Updated to unset interfaceref with deviation InterfaceRefConfigUnsupported as per your suggestion.